source/Agents/TrackingAgent.hpp

```diff
@@ -25,486 +27,486 @@
 
 namespace walle {
 
-/**
- * A single Alerter is responsible for forcefully changing the state of all trackers in its network
- * to TrackingState::TRACKING when a single TrackingAgent in the set of trackers
- * comes into range of its goal_pos
- */
-class Alerter {
- private:
-  /// World the agents in the agent network are a part of
-  cse491::WorldBase *world_ptr_;
-
-  /// Set of all agents in a single network
-  std::unordered_set<size_t> agent_network_set_;
-
- public:
-  // Must be constructed with an associated world
-  Alerter() = delete;
-  explicit Alerter(cse491::WorldBase *world_ptr);
-  Alerter(cse491::WorldBase *world_ptr, size_t id);
-  void AddAgent(size_t id);
-  void RemoveAgent(size_t id);
-  void AlertAllTrackingAgents(size_t caller_id);
-
-  /// Returns an immutable reference to the Alerter's set of agents it knows about
-  std::unordered_set<size_t> const& GetNetworkSet() const { return agent_network_set_; }
-}; // Alerter
+    /**
+     * A single Alerter is responsible for forcefully changing the state of all trackers in its network
+     * to TrackingState::TRACKING when a single TrackingAgent in the set of trackers
+     * comes into range of its goal_pos
+     */
+    class Alerter {
+    private:
+        /// World the agents in the agent network are a part of
+        cse491::WorldBase *world_ptr_;
+
+        /// Set of all agents in a single network
+        std::unordered_set<size_t> agent_network_set_;
+
+    public:
+        // Must be constructed with an associated world
+        Alerter() = delete;
+        explicit Alerter(cse491::WorldBase *world_ptr);
+        Alerter(cse491::WorldBase *world_ptr, size_t id);
+        void AddAgent(size_t id);
+        void RemoveAgent(size_t id);
+        void AlertAllTrackingAgents(size_t caller_id);
+
+        /// Returns an immutable reference to the Alerter's set of agents it knows about
+        std::unordered_set<size_t> const& GetNetworkSet() const { return agent_network_set_; }
+        }; // Alerter
```

Comment:
Let's change the indent of the Alerter class's ending brace so that it matches up with the indentation on line 35